### DISCUSSION TITLE
How to install Homebrew on Mavericks in 2021 ?

### DISCUSSION THREAD

**Discussion by spyderrp7** [Homebrew/discussions#2502] [Getting started]
I tried the command:

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

This command get's the error "Your OS X version is too old."

Which makes sense, since the last version of Homebrew that supports OS X Mavericks is 2.3.0.

At the moment the current version of Homebrew is 3.3.0. How can I install homebrew 2.3.0 which is compatible with OS X 10.9 Mavericks?

**Comment by SMillerDev** [comment]
Even if you manage to install it, I doubt it would work. But you can just use the https://docs.brew.sh/Installation#untar-anywhere method.

**Reply by spyderrp7** [reply]
You are right. I went a few steps forward with the untar method, but got so many errors... I give up!

**Reply by neutric** [reply]
[MacPorts](https://macports.org/) supports older macOS versions, their [guide](https://guide.macports.org/) explicitly mentions installing Xcode on 10.9 and [releases](https://github.com/macports/macports-base/releases/) exist for OS versions down to 10.4. Just in case you weren't aware and this might scratch your itch as well.
